src/components/translation/TranslationList.tsx

```diff
@@ -45,7 +45,7 @@ const TranslationList = () => {
       icon: '🇫🇷'
     },
     {
-      lang: 'hi',
+      lang: 'hin-IND',
```

Comment:
Did you read the translation guideline?

Reply:
"Hin" stands for hindi and "IND" stands for india (mostly everywhere) but i as per ["this page"](https://docs.oracle.com/cd/E23824_01/html/E26033/glset.html) its hi-IN. Donno its matter or not but should i change it ?

Reply:
Okay